src/main/python/ayab/engine/options_gui.ui

```diff
@@ -244,7 +244,8 @@
     <item>
      <widget class="QLabel" name="auto_mirror_icon">
       <property name="pixmap">
-       <pixmap resource="lowercase_e_reversed_rc.qrc">:/garamond-lowercase-e-reversed.png</pixmap>
+      </property>
```

Comment:
Possible regression?

Reply:
Resources are handled differently- if you load the app you'll see this still renders correctly.